### ISSUE TITLE
Application Error: App Broken

### ISSUE THREAD

**Issue by anburocky3** [jonandjess/joeschmoe.io#17]
![image](https://user-images.githubusercontent.com/15264938/209110403-c8f6f78c-588c-4b24-81c5-e29d8c001f55.png)

**Comment by xference** [comment]
> ![image](https://user-images.githubusercontent.com/15264938/209110403-c8f6f78c-588c-4b24-81c5-e29d8c001f55.png)

so did I.

**Comment by robetus** [comment]
Yeah, real bummer. This is the 3rd heroku app that has gone done on me in the last few months. There has to be something up with heroku. I'm considering hosting the app myself, it was really cool to have.

**Comment by crstauf** [comment]
Awhile back I created a Cloudflare Worker (with the owner's permission) that mimics the functionality of the app:
https://joeschmoe.crstauf.workers.dev/. It uses the same URL structure, without the preceding `api/v1`.

If you've forgotten the endpoints, the documentation can be accessed via the Wayback Machine.

**Comment by anburocky3** [comment]
> Awhile back I created a Cloudflare Worker (with the owner's permission) that mimics the functionality of the app: https://joeschmoe.crstauf.workers.dev/. It uses the same URL structure, without the preceding `api/v1`.
> 
> If you've forgotten the endpoints, the documentation can be accessed via the Wayback Machine.

How about writing the API docs, for that.

**Comment by crstauf** [comment]
@anburocky3 As I said, they are the same as joeschmoe.io, just without the `api/v1`.

**Comment by robetus** [comment]
I cloned the API/app for everyone to enjoy: https://joesch.moe/

**Comment by Awethentique** [comment]
> I cloned the API/app for everyone to enjoy: https://joesch.moe/

Thank you for this
